README.md

```diff
@@ -1,2 +1,54 @@
-# ASB_Tutorial
-Repository for 2020 American Society of Biomechanics Tutorial
+# ASB 2020 GitHub tutorial setup
+#### [Ryan Alcantara](https://twitter.com/Ryan_Alcantara_) & [Gary Bruening](https://twitter.com/garebearbru) & [Ross Wilkinson]
+
+This will be the repository that we will use for the Git Tutorial for [ASB 2020](https://asb2020.gatech.edu/)!
+
+We will keep updating this as more things get added.
+
+# Pre-Meeting Setup
+This section is intended to take you through the steps required before the meeting. 
+
+The first program everyone will need is some version of Git. Git is the version control software that allows us to see, track, and edit changes to code bases. 
+The second program is either [MATLAB](https://www.mathworks.com/) or [Python](https://www.python.org/). We will be using these to run out example scripts.
+
+## Installing Git
+This [website](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) has more detailed install instructions for all 3 operating systems.
+
+For this tutorial we will be focusing on using Git with [Github](https://github.com/). Github has produced a great desktop application for Windows and Mac that you may want to use.
+
+### Windows
+Windows has two options, you can install Git and use the Git Bash application, or you can use the Github desktop application.
+
+To download Git and use the Bash application, go to [this website](https://git-scm.com/download/win) and click the link associated with your version of Windows.
+
+To download the Github desktop application, go [here](https://desktop.github.com/) and download the application.
```

Comment:
Since the tutorial is focusing on the command line, I think it will be best to get attendees to use git bash. We can mention the github desktop app (and other version control GUIs) in the tutorial, but we want to avoid having some folks using the GUI and other using the command line.